src/clients/rest/rest_client.ts

```diff
@@ -31,8 +32,13 @@ class RestClient extends HttpClient {
 
     const link = ret.headers.get('link');
     if (params.query && link !== undefined) {
+      let limit = RestClient.DEFAULT_LIMIT;
```

Comment:
Nit: This feels a bit redundant. Can we accomplish this without the reassignment on this line?

Reply:
Agree. I think it looks better now